.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
php 5.3 CI'ı neden kaldırdık?

Reply:
Kaldırmadık abi, traviste php5.3 desteğini kaldırdığı için php 5.3 distribution değiştirdik @sotuzun ile beraber